model.yml

```
@@ -1,5 +1,5 @@
-short_name: template-model
-version: 1.0.0
-binary: project
+short_name: prostate
+version: 2.0.0
+binary: prostate
 config: 
-  - config/PhysiCell_settings.xml
+  - config/PhysiCell_settings_LNCaP_Luminespib_2.xml
```

```suggestion
  - config/PhysiCell_settings_LNCaP.xml
  - config/PhysiCell_settings_LNCaP_Luminespib_2.xml
  - config/PhysiCell_settings_LNCaP_Pictilisib_2.xml
  - config/PhysiCell_settings_LNCaP_Luminespib_Pictilisib_2.xml
```

This allows the different config files to be easily accessible

---

Ok, I thought you wanted "just" the default. Good!